### PR TITLE
Add struct-lmm

### PR DESCRIPTION
**Pull request by horta**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/struct-lmm/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    - pip
+    - pytest-runner
+    - python >3
+    - setuptools >=39.0.1
+  run:
+    - chiscore >=0.2.0
+    - glimix-core >=3.1.7
+    - numpy >=1.15
+    - numpy-sugar >=1.5.0
+    - pytest >=4
```
Comment (chrisburr) [inline recipes/struct-lmm/meta.yaml]: Is `pytest` needed as a runtime dependency?

File: recipes/struct-lmm/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    - pytest-doctestplus >=0.3.0
```
Comment (chrisburr) [inline recipes/struct-lmm/meta.yaml]: Similarly

File: recipes/struct-lmm/meta.yaml (hunk 1 above)
Comment (horta) [reply recipes/struct-lmm/meta.yaml]: Yep, I have built-in tests that uses pytest framework: `struct_lmm.test()`.

File: recipes/struct-lmm/meta.yaml (hunk 2 above)
Comment (horta) [reply recipes/struct-lmm/meta.yaml]: Same as above. That package allows for float comparison in doctests.

**Pull request merged**